### PR TITLE
Fix for void upgraded storage drawers using the previous developed void handling.

### PR DESCRIPTION
**Pull request by geldorn**
Also a fix for #251.

### CODE REVIEW

File: src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java [hunk 1]
```diff
@@ -41,47 +41,31 @@ private boolean isVoidable() {
...
+                    drawer.setStoredItemRedir(stack, inserted);
                 }
-            } else {
-                if (getStored() + size > drawer.getMaxCapacity(stack)) {
-                    int remainingSpace = getCapacity() - getStored();
-
-                    if (remainingSpace <= 0) {
-                        return isVoidable() ? null : ItemHandlerHelper.copyStackWithSize(stack, size);
-                    }
+                else {
```
Comment (raoulvdberge) [inline src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java]: coding style: else should be on same line

Comment (geldorn) [inline src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java]: It should be this is just looking strange in the diff.

Comment (raoulvdberge) [inline src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java]: It's not. Look at the file in your repo: https://github.com/geldorn/refinedstorage/blob/9c38c2998333cb4ab9a33d973ce29aa487bdfb9d/src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java#L53-L55

Comment (geldorn) [inline src/main/java/refinedstorage/tile/externalstorage/ItemStorageDrawer.java]: Was reading line and thinking column ;) Fixed.

**Comment by raoulvdberge** [comment]
Thank you!

**Pull request merged**